batch/batch/worker.py

```diff
@@ -692,6 +695,7 @@ async def delete(self):
     # }
     async def status(self):
         status = {
+            'version': INSTANCE_VERSION,
```

Comment:
So I think this shouldn't be the instance version.  The status data itself should have its own version, in the same way the spec has its own version.

Reply:
done